examples/pipelines/commoncrawl/components/extract_free_to_use_images/fondant_component.yaml

```diff
@@ -0,0 +1,25 @@
+name: Extract image url and license from commoncrawl
+description: Component that extracts image url and license metadata from a dataframe of webpage urls and html codes
+image: ghcr.io/ml6team/extract_image_licenses:latest
```

Comment:
can you rename the component folder to also be `extract_image_licenses` for consistency?